src/components/Animal.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+
+//import PropTypes from 'prop-types'
+
+export default function Animal({ title, picture }) {
+  return (
+    <AnimalStyled>
+      <img src={picture} alt="" width="100%" />
```

Comment:
Hier könntest du einen Alttext hinzufügen: "image of " und vllt kann man dann hier automatisch den Titel der Seite / Namen des Tieres angeben?